test/kmeans.jl

```diff
@@ -204,4 +204,11 @@ end
     end
 end
 
+@testset "get cluster assigments" begin
```

Comment:
Please also add the testset to `test/utils.jl` (it would be the new file that should be included from `runtests.jl` before all others) testing that `assign_clusters(.., R)` throws "not implemented" exception for an arbitrary `ClusteringResult` object other than `KmeansResult`, e.g. for `KMedoidsResult`.

Reply:
I´ve added the test